flows/msg.go

```diff
@@ -199,13 +201,14 @@ func (tc *TemplatingComponent) Preview(c assets.TemplateComponent) string {
 // MsgTemplating represents any substituted message template that should be applied when sending this message
 type MsgTemplating struct {
 	Template_   *assets.TemplateReference `json:"template"`
+	ExternalID_ string                    `json:"external_id"`
```

Comment:
We need this as for Twilio content template we need to send the template SID to the API and not the template name as we do not for WhatsApp API

Reply:
why can't mailroom just resolve that?

Reply:
Okay, I can move that to mailroom